service/dap/server.go

```diff
@@ -1552,8 +1552,8 @@ func (s *Server) onStackTraceRequest(request *dap.StackTraceRequest) {
 	// Determine if the goroutine is a system goroutine.
 	// TODO(suzmue): Use the System() method defined in: https://github.com/go-delve/delve/pull/2504
 	g, err := s.debugger.FindGoroutine(goroutineID)
-	var isSystemGoroutine bool
-	if err == nil {
+	isSystemGoroutine := true
+	if err == nil && g != nil {
```

Comment:
This code is about to be changed by this PR:
https://github.com/go-delve/delve/pull/2504/commits/c9578033995ea9b03beadfb0a2d98f31bfb7e52d
I think it got all approvals and is ready to go in. No merge conflicts.

Reply:
Ok looks great! I'll close this PR in favor of that one.